packages/story-editor/src/components/floatingMenu/layer.js

```diff
@@ -71,18 +100,18 @@ function FloatingMenuLayer() {
     // frame will already be updating because of the resize, so a DOM mutation is incoming.
   }, [workspaceWidth, workspaceHeight]);
 
+  const hasMenu = hasSelection && !isDismissed;
```

Comment:
One additional case to consider -- the menu should probably be hidden for edit mode of media, otherwise scaling slider will be on top of the edit menu. Or perhaps you were planning to do this in a media-specific PR?

Reply:
I don't see a menu in image edit mode. Do you? Edit-mode uses a different selection frame, which I haven't added the menu logic to, so it shouldn't be there at all.
![Screen Shot 2022-02-09 at 14 44 35](https://user-images.githubusercontent.com/637548/153279176-6431ef51-a43b-4193-9bab-b1240eae3686.png)

Reply:
Wait, there is a menu right there in the screenshot! How did I miss that? I'll take a look.